packages/atomic/src/index.html

```diff
@@ -117,8 +117,34 @@
 
           <atomic-no-results></atomic-no-results>
 
-          <!-- TODO: add result template example -->
-          <atomic-result-list></atomic-result-list>
+          <atomic-result-list fields-to-include="filetype,liboardinteractionstyle">
+            <!-- Custom result templates examples -->
+            <atomic-result-template
+              must-match-filetype="lithiummessage"
+              must-not-match-liboardinteractionstyle="contest"
+            >
+              <template>
+                <!-- Styling result template internaly -->
+                <style>
+                  h3 {
+                    font-weight: bold;
+                    color: var(--atomic-secondary);
+                  }
+                </style>
+                <h3>Contest message</h3>
+                <atomic-result-link></atomic-result-link>
+              </template>
+            </atomic-result-template>
+            <atomic-result-template id="singaporeTitle">
+              <template> 🇸🇬 <atomic-result-link></atomic-result-link> </template>
+              <!-- Adding Javascript conditions -->
+              <script>
```

Comment:
I guess this will be helped by reference docs but if you just start using atomic, I think it would be unclear what `conditions` is actually doing here.

Reply:
I understand this is mostly to help @jpmarceau  ? 

Im fine with adding this whole thing as a temporary "sample" I guess, but yeah, it's kind of random otherwise ;)

Reply:
Yes... index.html is mostly for devs playing with atomic, the more features we use on there, the more they will be able to understand

Reply:
We could very well comment these examples